src/compas_rhino/geometry/brep/brep.py

```diff
@@ -394,26 +404,38 @@ def trimmed(self, plane, tolerance=TOLERANCE):
         ----------
         plane : :class:`compas.geometry.Frame` or :class:`compas.geometry.Plane`
             The frame or plane to use when trimming. The discarded bit is in the direction of the plane's normal.
-
         tolerance : float
             The precision to use for the trimming operation.
 
+        Notes
+        -----
+        Trimming operation may result in multiple results (breps). When trimming, only one is used.
+        The used bit is the one on the opposite side of the cutting plane's normal.
+
         Returns
         -------
         :class:`~compas_rhino.geometry.RhinoBrep`
 
+        Raises
+        ------
+        BrepTrimmingError
+            If the trimming operation ended with no result.
+
+        See Also
+        --------
+        :meth:`compas_rhino.geometry.RhinoBrep.trim`
+
         """
         if isinstance(plane, Frame):
             plane = Plane.from_frame(plane)
         results = self._brep.Trim(plane_to_rhino(plane), tolerance)
-
-        breps = []
-        for result in results:
-            capped = result.CapPlanarHoles(TOLERANCE)
-            if capped:
-                result = capped
-            breps.append(RhinoBrep.from_native(result))
-        return breps
+        if not results:
+            raise BrepTrimmingError("Trim operation ended with no result")
+        result = results[0]
+        capped = result.CapPlanarHoles(TOLERANCE)
```

Comment:
in the signature, `tolerance=None`. then `tolerance = tolerance or TOL.absolute`, with at the top (if it is not already there) `from compas.tolerance import TOL`

Reply:
and then, finally, `capped = result.CapPlanarHoles(tolerance)`

Reply:
@tomvanmele done.
Just curious why not `tolerance=TOL.absolute` in signature? as it returns a `float` (immutable)

Reply:
that would have also been an option indeed. as long as it is not a hard coded tolerance value...